core/src/main/java/de/danoeh/antennapod/core/util/syndication/HtmlToPlainText.java

```diff
@@ -4,7 +4,7 @@
 
 import org.apache.commons.lang3.StringUtils;
 import org.jsoup.Jsoup;
-import org.jsoup.helper.StringUtil;
+import org.jsoup.internal.StringUtil;
```

Comment:
Hmm, `internal` sounds like we should use another method instead.

Reply:
Okay, let's keep it as an import. The method is only used in very few places anyway.